CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# How To Contribute
+There are several ways to contribute to the development of ReactOS. The most often encountered problem is not knowing where to begin or what to do. If you are able to program or understand the technical information that is pertinent to this project, helping the development can be easy.
+
+## Documentation
+
+There are some important points if you wouldd like to help document ReactOS:
+
+1.  Make sure the documentation does not exist yet (if it does, help improve it).
+2.  Respect [clean room reverse engineering] practices.
+3.  Add your knowledge to a place where the other developers can find it.
```

Comment:
This doesn't seem particularly actionable. Would it make sense to list examples of such places?